core/shared/platform/zephyr/zephyr_clock.c

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (C) 2024 Grenoble INP - ESISAR.  All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+ */
+
+#include "platform_api_extension.h"
+#include "platform_api_vmcore.h"
+
+#include "libc_errno.h"
+
+__wasi_errno_t
+os_clock_res_get(__wasi_clockid_t clock_id, __wasi_timestamp_t *resolution)
+{
+    return __WASI_ENOSYS;
```

Comment:
No clock related zephyr API to implement this function? Or could we implement it with k_uptime_get?

Reply:
I will look into this.

Reply:
I proposed some really basic implementation.